src/antares/model/study.py

```diff
@@ -171,6 +171,22 @@ def _directory_not_exists(local_path: Path) -> None:
     )
 
 
+def read_study_api(api_config: APIconf, study_id: str) -> "Study":
+    session = api_config.set_up_api_conf()
+    wrapper = RequestWrapper(session)
+    url = f"{api_config.get_host()}/api/v1/studies/{study_id}"
+    json_study = wrapper.get(url).json()
+
+    study_name = json_study.pop("name")
+    study_version = str(json_study.pop("version"))
+    json_study.pop("id")
+
+    study_settings = StudySettings(**json_study)
```

Comment:
To get the settings you should get the settings (see method _returns_study_settings inside `study_api.py`)

Also, you could add the `read_areas` method you introduced to add the areas directly.

The 2 remarks will need new unit tests and integration tests